### DISCUSSION TITLE
Port ConsoleTableApp to .NET 5.0

### DISCUSSION THREAD

**Discussion by vanillajonathan** [minhhungit/ConsoleTableExt#16] [Ideas]
Port the ConsoleTableApp from .NET Framework 3.5 to .NET 5.0. This would make it more modern and cross-platform since .NET 5.0 is officially available and supported on Linux and Mac.

Porting should be trivial.

**Comment by minhhungit** [comment]
@vanillajonathan Thank you for your suggestion 
ConsoleTableApp is just demo app, so no problem to upgrade it to .net 5, but the reason I keep it at 3.5 because we support .net framework 3.5 soI want to make sure that it works good with old frameworks
IMH upgrade to .NET 5 is not necessary
